cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/ZKBrokerFailureDetector.java

```diff
@@ -90,7 +90,7 @@ public void shutdown() {
   @Override
   Set<Integer> aliveBrokers() {
     // We get the alive brokers from ZK directly.
-    return CollectionConverters.asJava(_kafkaZkClient.getAllBrokersInCluster()).stream().map(Broker::id).collect(toSet());
+    return JavaConverters.asJavaCollection(_kafkaZkClient.getAllBrokersInCluster()).stream().map(Broker::id).collect(toSet());
```

Comment:
Should add a comment that it is deprecated in scala 2.13 but we change this as it's buggy?

To prevent future people from accidentally changing this again.

Reply:
Also I suppose this is unit-test-able?
But since this is a site up we can have another PR for it.

Reply:
And also state the symptom of the error (under scala xxx version, it never returns).

Reply:
I tried to create a unit test to use the same converter, and the test works. It seems to be a runtime only error.